torch/_inductor/codegen/simd.py

```diff
@@ -2034,9 +2034,37 @@ def convert_tiling_to_3d(
 
         if len(ranked_tilings) > 1:
             perf_hint_log.info("possibly bad tiling: %s", ranked_tilings)
+        # save iter vars in indirect broadcast memory reads
+        indirect_broadcast_pairs: list[tuple[sympy.Symbol, sympy.Symbol]] = []
+        if len(node_schedule) == 1:
```

Comment:
Can this restriction be loosen? What if there are fusion happening?

Reply:
not sure about this. let me try, see if there are ci errors and see if I can write a test.

Reply:
loose the restriction in https://github.com/pytorch/pytorch/pull/145992/commits/59e131647fe1bdc9a3c53b8643421629a44ff8fc let's see if this trigger some ci errors.